src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -443,9 +448,123 @@ def file_issue(testcase,
     else:
       raise
 
+  file_github_issue(testcase)
+
   # Update the testcase with this newly created issue.
   testcase.bug_information = str(issue.id)
   testcase.put()
 
   data_handler.update_group_bug(testcase.group_id)
   return issue.id, recovered_exception
+
+
+def get_github_access():
+  """Get access to GitHub with the oss-fuzz personal access token"""
+  token = db_config.get_value("oss_fuzz_robot_github_personal_access_token")
+  if not token:
+    logs.log_error("Unable to get oss-fuzz-robot's personal access token.")
+    return None
+  github_access = github.Github(token)
+  return github_access
+
+
+def file_github_issue(testcase):
+
+  def github_filing_enabled():
```

Comment:
Having too many nested functions like this makes the code a little hard to read. Let's just make them all top level functions in the github.py module.